plugins/organic/organic.cpp

```diff
@@ -269,30 +270,30 @@ void organicInstrument::playNote( NotePlayHandle * _n,
 			else
 			{
 				// create left oscillator
-				oscs_l[i] = new Oscillator(
+				 new (&oscs_l[i]) Oscillator(
 						&m_osc[i]->m_waveShape,
 						&m_modulationAlgo,
 						_n->frequency(),
 						m_osc[i]->m_detuningLeft,
 						static_cast<oscPtr *>( _n->m_pluginData )->phaseOffsetLeft[i],
 						m_osc[i]->m_volumeLeft,
-						oscs_l[i + 1] );
+						&oscs_l[i + 1] );
 				// create right oscillator
-				oscs_r[i] = new Oscillator(
+				 new (&oscs_r[i]) Oscillator(
 						&m_osc[i]->m_waveShape,
 						&m_modulationAlgo,
 						_n->frequency(),
 						m_osc[i]->m_detuningRight,
 						static_cast<oscPtr *>( _n->m_pluginData )->phaseOffsetRight[i],
 						m_osc[i]->m_volumeRight,
-						oscs_r[i + 1] );
+						&oscs_r[i + 1] );
 			}
 			
 				
 		}
 
-		static_cast<oscPtr *>( _n->m_pluginData )->oscLeft = oscs_l[0];
-		static_cast<oscPtr *>( _n->m_pluginData )->oscRight = oscs_r[0];
+		static_cast<oscPtr *>( _n->m_pluginData )->oscLeft = &oscs_l[0];
+		static_cast<oscPtr *>( _n->m_pluginData )->oscRight = &oscs_r[0];
```

Comment:
`&oscs_l[0]` and `&oscs_r[0]` can be written as `oscs_l` and `oscs_r` respectively.